debian/changelog

```diff
@@ -1,3 +1,10 @@
+opx-nas-l3 (3.10.1+opx1) unstable; urgency=medium
+
+  * Bugfix: Mac for L3 ARP fail to relearn after flapping the
```

Comment:
Suggestion: L3 ARP MAC relearning fails after the port-channel is flapped and switch is reloaded.